### DISCUSSION TITLE
Mediation Support

### DISCUSSION THREAD

**Discussion by ratson** [admob-plus/admob-plus#269] [Fund Features]
Please post your interests for which Ad Network would like to be supported.

This is a low priority feature.

**Comment by CaturViriyananda** [comment]
Facebook is my most favored one. because it is world wide available, 2nd to Admob in popularity.

**Comment by EMI-INDO** [comment]
Some of my games still use the cordova admob free plugin, for the reason that they want to use mediation Facebook and Unity Ads

https://www.npmjs.com/package/cordova-plugin-admob-mediation

I will continue to donate little by little.
Until Mediation Support@cordova is released,
I will continue to make donations for bug fixes, etc.

### Sharing is beautiful.

**Comment by richwebapps** [comment]
Unity Ads and InMobi

**Comment by cordovalife** [comment]
Unity, Facebook and IronSource please 🙏 i've been using the plugin and i'm very happy with it but admob ecpm is very low

**Comment by EMI-INDO** [comment]
@ratson 

Just want to know

how much money to fund this request

If someone wants to fund maybe I can help a little.

thank you.

**Reply by aymank900** [reply]
i'm willing to pay for Mediation Support 100$

**Comment by tuzke** [comment]
1000$

**Comment by Dharmesh2015** [comment]
facebook

**Comment by acemagic94** [comment]
facebook > unity. also support ad bidding will be great

**Comment by llapps** [comment]
Is it not enough to just integrate

framework src="com.google.ads.mediation:facebook:6.8.0.0" 
framework src="com.google.ads.mediation:applovin:11.1.3.0"
framework src="com.google.ads.mediation:adcolony:4.6.5.0" 
framework src="com.google.ads.mediation:ironsource:7.2.0.0"

something like that?

**Reply by gasci** [reply]
Has there been any update regarding this topic?

**Comment by reinaldoborin** [comment]
Do we have any update about mediation support?

**Comment by DesignerApparelSales** [comment]
Unity ads would be nice

**Comment by Winamp9** [comment]
Hi everyone. Has anyone tried connecting network adapters for mediation in build.gradle or anywhere else? Does mediation work for anyone with this plugin?

**Reply by 1n3JgKl9pQ6cUMrW** [reply]
Never tried, but would be nice to have.

Not sure if this plug-in is still maintained, 'though.
